alerts/alert.go

```diff
@@ -0,0 +1,71 @@
+package alerts
+
+import "github.com/MohawkTSDB/mohawk/backend"
+
+const (
+	CLOSED_INTERVAL RangeIntervalType = iota
+	LEFT_CLOSED_INTERVAL
+	RIGHT_CLOSED_INTERVAL
+)
+
+const (
+	PENDING AlertState = iota
```

Comment:
in our implementation `metric-value > X [ or metric value < Y ] ` we only have `state = true or false`, I think we can do without this for now ( we can re-intruduce this when we implement it )